R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
Thanks a lot, @drknexus, although I don't think I got them to change their point of view :)
They were extremely helpful with trying to come up with a solution that's convenient for all parties, and after a couple of e-mails this suggestions was accepted. Now the package is very small (only a few kilobytes), so the archive footprint is negligible, although the binary packages on CRAN still include the jar files, so an overall ~100 MB extra storage -- but I hope many R packages will be able to build on the top of that.

BTW do you plan to attend http://unconf17.ropensci.org? I think there will be some R/AWS-related fun, see eg https://twitter.com/sjackman/status/830536912376967169